pkg/sysregistriesv2/system_registries_v2.go

```diff
@@ -387,6 +425,14 @@ func (config *V2RegistriesConf) postProcessRegistries() error {
 			if mir.Location == "" {
 				return &InvalidRegistries{s: "invalid condition: mirror location is unset"}
 			}
+
+			if reg.MirrorByDigestOnly && mir.PullFromMirror != "" {
+				return &InvalidRegistries{s: "cannot set mirror usage mirror-by-digest-only for the registry and pull-from-mirror for per-mirror at the same time"}
```

Comment:
Non-blocking: It would be useful to identify the problematic registry/mirror (via `reg.Prefix`/`mir.Location`) in the error message.